dojo_plugin/api/v1/search.py

```diff
@@ -59,7 +59,7 @@ def get(self):
                     {
                         "id": dojo.reference_id,
                         "name": dojo.name,
-                        "link": f"/{dojo.reference_id}",
+                        "link": f"/dojo/{dojo.reference_id}",
```

Comment:
Reminder for myself: Restore the link format. Breaking change for original deployment